client/src/mapView.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { Map, TileLayer } from 'react-leaflet';
+import ReactModal from 'react-modal';
+import { tileSet, SFGeo, zoomLevel, mapAttribution } from '../mapconfig';
+import MarkerCollection from './markerCollection';
+
+const PARKLETS_ENDPOINT = 'https://data.sfgov.org/resource/6a7x-cttf.json';
+
+const fetchData = (url) => {
+  const fetched = fetch(url)
```

Comment:
is there a fetch module? Also fetch is async you unless you async/await, you `fetched` will be undefined since it won't resolve before return

Reply:
No fetch module imported.  I never got an undefined value when running in the browser nor from the linter nor tests.  The return value is a _promise_.  I suspect the render works because the function gets called in componentDidMount, so when the output comes back as resolved, and setState gets called, the component re-renders.

Reply:
Ah seems fetch is build into React. I'd say to return the promise after all the work is done instead of declaring it to a var. Its not really needed to declare since its going to be used by an external function.